packages/paste-website/src/pages/form-elements/textarea/index.mdx

```diff
@@ -0,0 +1,604 @@
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/form"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/form-elements/textarea/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Textarea"
+  categoryRoute={SidebarCategoryRoutes.FORM}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/form"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-form--textarea-options"
+  abstractUrl="https://share.goabstract.com/6cc5fc40-3408-4cfd-b1d0-103aea01601d"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+A textarea allows users to enter text on multiple lines.
+
+### About Textarea
+
+Use a textarea when you want to allow a user to enter text on multiple lines. A textarea allows for any text to be entered and can't be restricted as [inputs](/form/input/) can.
+
+### Accessibility
+
+<UnorderedList>
+  <ListItem>Include a visible label on <strong>all</strong> form fields.</ListItem>
+  <ListItem>Each lable must use the <inlineCode>htmlFor</inlineCode> prop that equals the <inlineCode>id</inlineCode> of the associated textarea.</ListItem>
+  <ListItem>
+    Don't use placeholder text as a replacement for labels. It can be used to provide an example to users, but will
+    disappear from the field when a user enters text. It's also not broadly supported by assistive technologies and
+    won't display in older browsers.
+  </ListItem>
+  <ListItem>
+    Use one of 3 ways to label a form field:
+    <UnorderedList marginBottom="space0">
+      <ListItem>Visible label with <Anchor href="#formlabel-props">FormLabel</Anchor> (preferred)</ListItem>
+      <ListItem>Visible label that's associated to the textarea with <inlineCode>aria-labelledby</inlineCode></ListItem>
+      <ListItem>Label directly using <inlineCode>aria-label</inlineCode></ListItem>
+    </UnorderedList>
+  </ListItem>
+  <ListItem>
+    Provide clear identification of required fields in the label or at the start of a form. If you use the required
+    field indicator, include the form key at the start of the form.
+    <UnorderedList marginBottom="space0">
+      <ListItem>Use the <inlineCode>required</inlineCode> prop to programatically indicate they are required to browsers.</ListItem>
+    </UnorderedList>
+  </ListItem>
+  <ListItem>
+    Include inline error text with the error icon on any field that errors to make it visually clear that the field
+    changed.
+  </ListItem>
+  <ListItem>If the textarea has assicated help text or error text, include the <inlineCode>aria-describedby</inlineCode> prop on the textarea. This should match the <inlineCode>id</inlineCode> of the helper or error text.</ListItem>
```

Comment:
assicated > associated

Reply:
and a space between described by

Reply:
Fixed the assicated error. Leaving the describedby. That is the correct name for the attribute: `aria-describedby`.